assets/src/styles/components/_header.scss

```diff
@@ -21,9 +22,10 @@
 	.HMLogo {
 
 		display: block;
-		width: 240px;
-		height: 46px;
-		margin: #{ .75 * $base-line-height } auto $base-line-height;
+		width: 210px;
+		height: 40px;
+		margin: 7px auto; // 7px =  ( 2rem - 40px ) / 2
```

Comment:
Can we use maths here rather than magic number, stuff has been calculated either with sass or calc elsewhere?